pkgs/servers/microwindows/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, automake, zlib, libpng, libjpeg
+ , freetype, libX11, libXext, yacc, flex }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2019-05-20";
+  pname = "microwindows";
+
+  src = fetchFromGitHub {
+    owner = "ghaerr";
+    repo = "microwindows";
+    rev = "ad3164ddd42fb28761a04ec69534e4486a5894f8";
+    sha256 = "0qddkzwax96i0g73ir378s37b3xhihpz2146a3yph4l2p6lcdd8i";
+  };
+
+  sourceRoot = "source/src";
+
+  patchPhase = ''
+    cp Configs/config.linux-X11 config
+    # canceled failing demo
+    substituteInPlace demos/nuklear/Makefile --replace "all:" "#all:" 
+  '';
+
+  nativeBuildInputs = [ automake ];
+  buildInputs = [ libpng libjpeg freetype libX11 libXext yacc flex zlib ];
+
+  makeFlags = [ 
+    "LEX=flex"
+    "HAVE_JPEG_SUPPORT=Y"
+    "INSTALL_PREFIX=$(out)"
+    "INSTALL_OWNER1= INSTALL_OWNER2="
+  ];
+
+  # copy missing binaries
+  postInstall = ''
+    cp bin/* $out/bin
+  '';
+
+  postFixup =
```

Comment:
Please be consistent in style and spacing. Specifically:
- -> `postFixup = ''` (same line, like the rest of your code)
- `patchelf` is indented only 1 space instead of 2 above `find`